app/Exceptions/DockerNotAvailableException.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Exceptions;
+
+use App\Shell\Environment;
+use App\Shell\Shell;
+use Exception;
+
+class DockerNotAvailableException extends Exception
+{
+    public function render($request = null): void
+    {
+        $console = app('console');
+        $shell = app(Shell::class);
+
+        $console->line('');
+        $console->line($shell->formatErrorMessage('Docker is not available.'));
+        $console->line('');
+
+        if (in_array(PHP_OS_FAMILY, ['Darwin', 'Linux', 'Windows'])) {
+            $osSpecificHelp = 'helpFor' . ucfirst(PHP_OS_FAMILY);
+            $this->$osSpecificHelp();
+        }
```

Comment:
@mattstauffer I'm not sure this is much cleaner but it's at least separated by environment. I'm open to any suggestions!

Reply:
I like it!